scripts/healthcheck.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
Good point. I agree, a `yarn health` that calls `yarn lint && yarn build && yarn todo` seems good enough. Then we can keep the husky file as is.

Reply:
Can't we just use the standard `healthcheck.js` from `project-templates` here? customising this for one repo only is configuration drift.

Reply:
I'd ditch it because

1. it doesn't provide any value here. We can do `yarn health` as described above without the need of a `healthcheck.js`
2. IMO fine to deviate from our `project-templates`. Why is it relevant to enforce our `bbk init` setup here where 1. it's an open source project and 2. it's typescript which causes many deviations by itself already

Reply:
it just makes a lot of things easier to stick to a standard. this won't be our last open source project. and if we want to use typescript internally, we will create a healthcheck for it anyways. 

regarding ditching this file: we used to do just `yarn lint:bla && yarn lint:blub` but the sentiment then was that it was a very noisy task to run locally (where it's primarily run) and made it difficult to see what went wrong and where. which was the reason we came up with the clean formatting in the first place.

Reply:
Do you know why we have JS and shell healthchecks? The shell-based one with `chronic` seems useful and I agree with here with your point of better output. What's the purpose of `healthcheck.js` though?

Reply:
I believe the main reason for switching to a JS-based approach was that you don't have any dependency issues. If it was a bash script, every developer needs to locally have `chronic` available AND we'd need to include the `chronic` binary in our CI image which comes with its own overhead.

So we said, hey why don't we make this just another `devDependency` same as all other dev tooling.

Reply:
Thanks.

No strong opinion. From my side:
- `chronic` won't be required for users, just for devs. This is us and eventual external contributors. Seems fine, especially with making this part of `CONTRIBUTION.md`.
- `chronic` is not required in any images because we're never running `yarn health` in CI - only locally.
- Having a `healthcheck.js` in a typescript project seems off. Would favor `healthcheck.ts` that is called with `ts-node`.

Please let me know your choice.

Reply:
I think the amount of code we're talking about here is fairly minimal, so IMO paying a few more lines of code for 
1) Getting rid of `chronic` as an external dependency
2) Keeping everything in JS

is a good tradeoff.

If you want to refactor it to TypeScript, that's fine with me. However, a simple `node scripts/healthcheck.js` will then no longer suffice and we need some kind of typescript binary. (Does something like `ts-node` come bundled with the `typescript` npm package?)

Reply:
done